### PR TITLE
Using dropdown to select the language and localized to Brazilian Portuguese 

### PR DESCRIPTION
**Pull request by rayronvictor**
My purpose is to provide a way to localize to more than two languages.

### CODE REVIEW

File: src/locales/zh-CN.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // import zhMessages from '../../locales/zh.json';
 
 export default {
-  'navbar.lang': 'English',
```
Comment (chenshuai2144) [inline src/locales/zh-CN.js]: This is intentional and does not need to be changed.

File: src/locales/pt-BR.js [hunk 2]
```diff
@@ -0,0 +1,189 @@
+export default {
+  'navbar.lang': 'Português',
```
Comment (chenshuai2144) [inline src/locales/pt-BR.js]: Can you change to 'English' ?
Convenient for people who don't know Português to switch to a familiar language

Comment (rayronvictor) [reply src/locales/pt-BR.js]: People need to know how to change the language and with "Português" do not understand that there is the place to change? Is that what you mean?

Currently it looks like this:
captura de tela 2018-09-25 as 07 57 19

What you think about this (from umijs.org):
captura de tela 2018-09-25 as 08 01 18

And changing the locale file to `'navbar.lang': 'Languages'`

Comment (chenshuai2144) [reply src/locales/pt-BR.js]: you are right

**Comment by chenshuai2144** [comment]
Still have some conflicts

**Comment by chenshuai2144** [comment]
rebase master？

**Pull request merged**